x/gov/migrations/v046/json.go

```diff
@@ -30,5 +30,6 @@ func MigrateJSON(oldState *v1beta1.GenesisState) (*v1beta2.GenesisState, error)
 		DepositParams:      &depParams,
 		VotingParams:       &votingParms,
 		TallyParams:        &tallyParams,
+		BurnParams:         v1beta2.DefaultGenesisState().BurnParams,
```

Comment:
we actually also need to add store.go in-place sotre migrations

Reply:
add in place for deposit params, would love a check before writing tests

Reply:
LGTM. maybe we also need `paramstore.WithKeyTable(types.ParamKeyTable())`, but not sure.